Tests/SKCoreTests/ToolchainRegistryTests.swift

```diff
@@ -36,36 +36,6 @@ final class ToolchainRegistryTests: XCTestCase {
     await assertTrue(tr.default === tr.toolchain(identifier: "a"))
   }
 
-  func testDefaultDarwin() async throws {
-    let prevPlatform = Platform.current
```

Comment:
These tests are no longer supported because `Platform.current` isn’t mutable anymore. We should still have coverage fro this because we’re running tests on macOS and Linux.